tree/dataframe/test/dataframe_helpers.cxx

```diff
@@ -121,61 +121,46 @@ class SaveGraphTestHelper {
+             ">, style=\"filled\", fillcolor=\"" + (hasLoopRun ? "#e6e5e6" : "#e47c7e") +
+             "\", shape=\"box\"];\n\t6 [label=<Filter>, style=\"filled\", fillcolor=\"#0f9d58\", "
+             "shape=\"hexagon\"];\n\t7 [label=<Define<BR/>Branch_1_1_def>, style=\"filled\", fillcolor=\"#4285f4\", "
+             "shape=\"ellipse\"];\n\t3 [label=<Filter>, style=\"filled\", fillcolor=\"#0f9d58\", "
+             "shape=\"hexagon\"];\n\t4 "
+             "[label=<Define<BR/>Branch_1_def>, style=\"filled\", fillcolor=\"#4285f4\", shape=\"ellipse\"];\n\t5 "
+             "[label=<Define<BR/>Root_def2>, style=\"filled\", fillcolor=\"#4285f4\", shape=\"ellipse\"];\n\t1 "
+             "[label=<Filter>, style=\"filled\", fillcolor=\"#0f9d58\", shape=\"hexagon\"];\n\t2 "
+             "[label=<Define<BR/>Root_def1>, style=\"filled\", fillcolor=\"#4285f4\", shape=\"ellipse\"];\n\t0 "
+             "[label=<Empty source<BR/>Entries: 8>, style=\"filled\", fillcolor=\"#f4b400\", shape=\"ellipse\"];\n\t11 "
+             "[label=<Count" +
+             (hasLoopRun ? "<BR/><FONT POINT-SIZE=\"10.0\">Already Run</FONT>" : "") +
+             ">, style=\"filled\", "
+             "fillcolor=\"" +
+             (hasLoopRun ? "#e6e5e6" : "#e47c7e") +
+             "\", "
+             "shape=\"box\"];\n\t9 [label=<Filter>, style=\"filled\", fillcolor=\"#0f9d58\", shape=\"hexagon\"];\n\t10 "
+             "[label=<Define<BR/>Branch_1_2_def>, style=\"filled\", fillcolor=\"#4285f4\", shape=\"ellipse\"];\n\t17 "
+             "[label=<Max" +
+             (hasLoopRun ? "<BR/><FONT POINT-SIZE=\"10.0\">Already Run</FONT>" : "") +
+             ">, style=\"filled\", fillcolor=\"" + (hasLoopRun ? "#e6e5e6" : "#e47c7e") +
+             "\", "
+             "shape=\"box\"];\n\t14 [label=<Filter>, style=\"filled\", fillcolor=\"#0f9d58\", "
+             "shape=\"hexagon\"];\n\t15 "
+             "[label=<Define<BR/>Branch_2_2_def>, style=\"filled\", fillcolor=\"#4285f4\", shape=\"ellipse\"];\n\t16 "
+             "[label=<Define<BR/>Branch_2_1_def>, style=\"filled\", fillcolor=\"#4285f4\", shape=\"ellipse\"];\n\t12 "
+             "[label=<Filter>, style=\"filled\", fillcolor=\"#0f9d58\", shape=\"hexagon\"];\n\t13 "
+             "[label=<Define<BR/>Branch_2_def>, style=\"filled\", fillcolor=\"#4285f4\", shape=\"ellipse\"];\n\t18 "
+             "[label=<Count" +
+             (hasLoopRun ? "<BR/><FONT POINT-SIZE=\"10.0\">Already Run</FONT>" : "") +
+             ">, style=\"filled\", "
+             "fillcolor=\"" +
+             (hasLoopRun ? "#e6e5e6" : "#e47c7e") +
+             "\", "
+             "shape=\"box\"];\n\t6 -> 8;\n\t7 -> 6;\n\t3 -> 7;\n\t4 -> 3;\n\t5 -> 4;\n\t1 -> 5;\n\t2 -> 1;\n\t0 -> "
+             "2;\n\t9 "
+             "-> 11;\n\t10 -> 9;\n\t4 -> 10;\n\t14 -> 17;\n\t15 -> 14;\n\t16 -> 15;\n\t12 -> 16;\n\t13 -> 12;\n\t5 -> "
+             "13;\n\t13 -> 18;\n}";
```

Comment:
This is better implemented with string interpolation, i.e. using ROOT's `Form` function (that returns a `TString`, which can be converted to a std::string). See doxygen or e.g. https://root-forum.cern.ch/t/variable-title-mutiple-lines/48625/2?u=eguiraud